type-definitions/ts-tests/es6-collections.ts

```diff
@@ -1,18 +1,23 @@
-import {
-  Map as ImmutableMap,
-  Set as ImmutableSet,
-} from 'immutable';
+import { expect, test } from 'tstyche';
+import { Map as ImmutableMap, Set as ImmutableSet } from 'immutable';
 
-// Immutable.js collections
-const mapImmutable: ImmutableMap<string, number> = ImmutableMap<string, number>();
-const setImmutable: ImmutableSet<string> = ImmutableSet<string>();
+test('immutable.js collections', () => {
+  const mapImmutable: ImmutableMap<string, number> = ImmutableMap<
+    string,
+    number
+  >();
+  const setImmutable: ImmutableSet<string> = ImmutableSet<string>();
 
-// $ExpectType Map<string, number>
-mapImmutable.delete('foo');
```

Comment:
Here this is `ImmutableMap` not `Map`. Seems like `dtslint` is not aware of the context.

Reply:
If I understood dtslint, it does compare the string output, so like it is in the `immutable.d.ts` file, so `Map` and not `ImmutableMap`

Reply:
Yes. It works correctly, only looks strange.